src/ucp/core/ucp_request.inl

```diff
@@ -807,4 +807,22 @@ ucp_request_invoke_uct_completion(ucp_request_t *req, ucs_status_t status)
     ucp_invoke_uct_completion(&req->send.state.uct_comp, status);
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_request_complete_recv_rndv(ucp_request_t *req, ucs_status_t status)
+{
+    if (req->flags & UCP_REQUEST_FLAG_RECV_AM) {
+        ucp_request_complete_am_recv(req, status);
+    } else {
+        ucs_assert(req->flags & UCP_REQUEST_FLAG_RECV_TAG);
+        ucp_request_complete_tag_recv(req, status);
+    }
+}
+
+static UCS_F_ALWAYS_INLINE void
+ucp_request_complete_recv_rndv_common(ucp_request_t *rreq, ucs_status_t status)
+{
+    ucp_request_recv_buffer_dereg(rreq);
+    ucp_request_complete_recv_rndv(rreq, status);
+}
```

Comment:
seems these are used in rndv.c only (no need to expose), but there are the same funcs there:
`ucp_rndv_recv_req_complete`
`ucp_rndv_zcopy_recv_req_complete`